pkg/cmd/release/create/create.go

```diff
@@ -469,17 +472,17 @@ func createRun(opts *CreateOptions) error {
 	return nil
 }
 
-func gitTagInfo(tagName string) (string, error) {
-	cmd, err := git.GitCommand("tag", "--list", tagName, "--format=%(contents:subject)%0a%0a%(contents:body)")
+func gitTagInfo(client *git.Client, tagName string) (string, error) {
```

Comment:
Should this operations eventually be methods on the `git.Client` itself?

Reply:
Yeah that makes sense to me. Will leave that for a followup PR.